src/HypothesisClient/HttpClient/HttpClientInterface.php

```diff
@@ -1,11 +1,11 @@
 <?php
 
-namespace eLife\HypothesisClient;
+namespace eLife\HypothesisClient\HttpClient;
 
 use GuzzleHttp\Promise\PromiseInterface;
 use Psr\Http\Message\RequestInterface;
 
-interface HttpClient
+interface HttpClientInterface
```

Comment:
I'll look at extracting this into a separate library, as it could be shared between our API client and the Hypothesis one (so a library for JSON APIs).